evap/staff/templates/staff_questionnaire_form.html

```diff
@@ -49,9 +49,18 @@ <h5 class="card-title">{% trans 'Questions' %}</h5>
                                     {% else %}
                                         <td></td>
                                     {% endif %}
-                                    {% for field in form_element.visible_fields %}
-                                        <td>{% include 'bootstrap_form_field_widget.html' with field=field %}</td>
-                                    {% endfor %}
+                                    <td>{% include 'bootstrap_form_field_widget.html' with field=form_element.text_de %}</td>
+                                    <td>{% include 'bootstrap_form_field_widget.html' with field=form_element.text_en %}</td>
+                                    <td class="question-type">
+                                        {% include 'bootstrap_form_field_widget.html' with field=form_element.type %}
+                                        <div class="form-check mt-1">
+                                            <input class="form-check-input allow-textanswer-checkbox" id="{{ form_element.allows_additional_textanswers.id_for_label }}" name="{{ form_element.allows_additional_textanswers.html_name }}"
+                                                type="checkbox"{% if form_element.allows_additional_textanswers.value %} checked{% endif %}
+                                                {% if form_element.allows_additional_textanswers.field.disabled %} disabled{% endif %} />
```

Comment:
It looks like you’re mirroring some basic form logic. In a little test, it worked fine for me to include in the `forms.py` the additional attribute as `widgets = {…, 'allows_additional_textanswers': forms.CheckboxInput(attrs={'data-keep': 'checked'})}` and replacing the `<input>…</input>` with just `{{ form_element.allows_additional_textanswers }}`.
Or was there a different reason I don’t see?

Reply:
yes, it works in part. but unfortunately, the label then isn't caught by the `:disabled` anymore, and the styling is broken.
so this needs additional changes.

Reply:
it turns out this is more complicated than i thought and not worth it for me right now. will leave it as is.